SPARCore/GADRemoteModel.m

```diff
@@ -1,15 +1,20 @@
 #import "GADRemoteModel.h"
 
-@implementation GADRemoteModel
+static NSString *const API_HOSTNAME = @"https://g2j7qs2xs7.execute-api.us-west-2.amazonaws.com/";
+static NSString *const API_PREFIX = @"devstable";
+
+static NSString *const API_ITEMS = @"items";
+static NSString *const API_NEXT_PAGE_TOKEN = @"nextPageToken";
 
 const NSTimeInterval timeoutInterval = 60.0;
 
+@implementation GADRemoteModel
+
 + (void) fetchModelsWithParams:(NSURL * _Nonnull)baseURL
```

Comment:
Should this method take a `token` or how will they be able to use the one they get back?